ui/sender/sender.go

```diff
@@ -455,11 +450,14 @@ func (m *model) copyReceiverCommand() string {
 	var builder strings.Builder
 	builder.WriteString("portal receive ")
 	builder.WriteString(m.password)
-	for flag, value := range m.copyFlags {
+
+	relayAddrKey := "relay"
+	if !config.IsDefault(relayAddrKey) {
 		builder.WriteRune(' ')
-		builder.WriteString(flag)
+		builder.WriteString(fmt.Sprintf("--%s", relayAddrKey))
 		builder.WriteRune(' ')
-		builder.WriteString(value)
+		builder.WriteString(viper.GetString(relayAddrKey))
 	}
+
```

Comment:
Maybe is overkill, but what do you think of having a config option for disabling the behavior of adding flags to the copy command?

Reply:
It feels a bit overkill/specific but I had the same thought. Decided to leave it like this but we can definitely extend the Config with stuff like that in the future.